packages/lib/versionInfo.test.ts

```
@@ -128,3 +129,38 @@ describe('getPluginLists', () => {
+
+	it('should show Rich Text editor when codeView is false', () => {
+		const original = Setting.value('editor.codeView');
+		Setting.setValue('editor.codeView', false);
+		try {
+			const v = versionInfo(packageInfo, {});
+			expect(v.body).toContain('Editor: Rich Text');
+		} finally {
+			Setting.setValue('editor.codeView', original);
+		}
+	});
+});
```

ESLint

[error] There should not be more than 1 describe at the top level

(jest/require-top-level-describe)